target/linux/ath79/nand/config-default

```diff
@@ -1,15 +1,27 @@
-CONFIG_BCH=y
-CONFIG_MTD_NAND=y
+CONFIG_BLK_MQ_PCI=y
+CONFIG_LEDS_RESET=y
 CONFIG_MTD_NAND_AR934X=y
```

Comment:
Support of the AR934X driver will likely require the following changes to the kernel config. 

BCH support does _not_ appear needed at this time, based on 
```
target/linux/ath79/dts/ar934x.dtsi:			nand-ecc-mode = "hw";
```

Awaiting confirmation from @realmicu, or someone with a device in hand, with a deeper knowledge of the driver and device.
```
--- a/target/linux/ath79/nand/config-default
+++ b/target/linux/ath79/nand/config-default
@@ -1,7 +1,9 @@
 CONFIG_BLK_MQ_PCI=y
 CONFIG_LEDS_RESET=y
+CONFIG_MTD_NAND=y
 CONFIG_MTD_NAND_AR934X=y
 CONFIG_MTD_NAND_CORE=y
+CONFIG_MTD_NAND_ECC=y
 CONFIG_MTD_SPI_NAND=y
 CONFIG_MTD_UBI=y
 CONFIG_MTD_UBI_BLOCK=y
```

Reply:
Resolved in 2019-10-22 push

Reply:
Changes now present in 2019-10-22 push